pkg/loadbalancer/bpf_abi.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Is the plan to include this commit or to merge https://github.com/cilium/cilium/pull/12436?

Reply:
Depends on whether we want to avoid cgo.

If we don't mind using cgo, we can merge this as it is, and forget #12436.

If we want to avoid cgo, I would suggest the following: remove the second commit  from this PR (i.e,. the `bpf_abi` test), so that it can be merged; and, add a test for the constants in this code as a subsequent commit to #12436.

Apologies for any confusion.